docker/docker-compose.yml

```diff
@@ -22,6 +22,8 @@ services:
       - redislocal
   redislocal:
     image: redis
+    # Uncomment this if you don't want the redis data to persist
+    #command: "redis-server --save '' --appendonly no"
```

Comment:
Oh and this little line was helpful for me to ensure that redis wasn't persisting between testing runs, as it was getting a strange no space left on device error. My computer has plenty of room, so I'm not sure where that is coming from all of a sudden.

Reply:
check the docker settings for how much space is allowed for all containers? I had to bump mine up a few times.

Reply:
Ah, I didn't even know that was a setting. Thanks!